.github/pull_request_template.md

```diff
@@ -0,0 +1,23 @@
+<!-- Please use this template for your pull request. -->
+<!-- Please use the sections that you need and delete other sections -->
+
+## This PR
+<!-- add the description of the PR here -->
+
```

Comment:
Nit: swap these two lines to be consistent with Related Issues

Reply:
done